frontend/pulse3d/src/pages/upload-form.js

```diff
@@ -599,9 +611,9 @@ export default function UploadForm() {
               options={formattedUploads}
               width={500}
               label="Select Recording"
-              reset={files.length === 0}
+              reset={resetFilesDropDown && files.length === 0}
               handleSelection={handleDropDownSelect}
-              defaultFile={defaultReanalysisFile}
+              defaultIndex={formattedUploads.indexOf(defaultReanalysisFile)}
```

Comment:
It would probably be better to just pass in the default option directly here

Reply:
I would still need the index to call `handleSelection`. So that would be adding in extra prop. Would it be ok to keep it as is?